pkg/checkpoint/process_test.go

```diff
@@ -200,7 +201,8 @@ func TestProcess(t *testing.T) {
 				},
 				"AA": {},
 			},
-			expectStop:        []string{"AA", "kube-system/pod-checkpointer"},
+			expectStart:       []string{"kube-system/pod-checkpointer"},
```

Comment:
Same reason as above for the checkpointer starting. We still stop AA because we stop a stop-then-remove.